econml/dml/dml.py

```diff
@@ -642,6 +657,12 @@ class LinearDML(StatsModelsCateEstimatorMixin, DML):
         If None, the random number generator is the :class:`~numpy.random.mtrand.RandomState` instance used
         by :mod:`np.random<numpy.random>`.
 
+    use_ray: bool, default False
+        Whether to use Ray to parallelize the cross-fitting step. If True, Ray must be installed.
```

Comment:
Can you fix the spacing here with a new line in between the arg descriptions. Same for SparseLinearDML and KernelDML

Reply:
Noted